src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -216,6 +231,41 @@ static bool ShouldTreatAsVariadic(PInvoke[] candidates)
         }
     }
 
+    private static string FixupSymbolName(string name)
+    {
+        UTF8Encoding utf8 = new();
+        byte[] bytes = utf8.GetBytes(name);
+        StringBuilder sb = new();
+
+        foreach (byte b in bytes)
+        {
+            if ((b >= (byte)'0' && b <= (byte)'9') ||
+                (b >= (byte)'a' && b <= (byte)'z') ||
+                (b >= (byte)'A' && b <= (byte)'Z') ||
+                (b == (byte)'_'))
```

Comment:
So the most common character outside of this range are probably going to be `'.'` and `'+'` for nested `type!.FullName`. I wonder if we should mangle those in some shorter way.

Reply:
ugh.. that's the reason I had retained `    private static char[] s_charsToReplace = new[] { '.', '-', };`, but looks like I messed up the patch when rebasing, merging multiple times!

We could `'+'` to that array, and check that?

Reply:
I mean, if it's in `s_charsToReplace`, then use `_`?